XCUITest/CopyTest.swift

```diff
@@ -21,14 +21,18 @@ class CopyTest: BaseTestCase {
     func testCopyMenuItem() {
         let urlBarTextField = app.textFields["URLBar.urlText"]
         
-        loadWebPage("google.com")
-        urlBarTextField.press(forDuration: 1.0)
+        loadWebPage("facebook.com")
+        UIPasteboard.general.string = ""
+        urlBarTextField.coordinate(withNormalizedOffset: CGVector.zero).withOffset(CGVector(dx:10,dy:0)).press(forDuration: 1.5)
```

Comment:
It's probably a good idea to write a comment as to why we're doing this weird coordinate offset thing :)